storage/innobase/xtrabackup/src/backup_mysql.h

```diff
@@ -113,4 +113,16 @@ mdl_lock_table(ulint space_id);
 void
 mdl_unlock_all();
 
+bool
+has_innodb_buffer_pool_dump();
```

Comment:
Consider making it static and local to `backup_mysql`, `check_dump_buffer_pool` could be made the only entry point.